src/FileSystem-Tests-Core/FileSystemTest.class.st

```diff
@@ -679,6 +679,23 @@ FileSystemTest >> testRootIsNotAFile [
 	self deny: (filesystem isFile: Path root)
 ]
 
+{ #category : #'tests-streams-compatibility' }
+FileSystemTest >> testTooManyDirectoryDescriptorOpened [
+	| opened testSize |
+
+	self halt.
+	testSize := 1025.
+	opened := OrderedCollection new:testSize.
+
+	self should: 
+		[1 to: testSize do: [ :x | opened add: ((File primOpendir: '/etc') at:3)]] 
+	raise: Error.
+	
+	opened do:[:each |
+    File primClosedir: each. ]
+
```

Comment:
This method requires some nice formatting...

Reply:
Too much crashes made me forget what version i was committing :)